Sources/Classes/PDFViewController.swift

```diff
@@ -163,6 +172,15 @@ public final class PDFViewController: UIViewController {
     }
 }
 
+//MARK: Customization
+extension PDFViewController {
+    public func setBackgroundColor(to color: UIColor) {
```

Comment:
Personally I don't feel like a function is the way this should be done in Swift3. I think the best way to achieve this is by having a property and setting the view & collection view background colors within the ```didSet```. For exmaple:

``` swift
var backgroundColor: UIColor = default {
   didSet {
      view.backgroundColor = backgroundColor
      collectionView.backgroundColor = backgroundColor
   }
}
```